### ISSUE TITLE
check box does not show up in time tracking

### ISSUE THREAD

**Issue by sammus9474** [ffalor/bird-run-alert#2]
I've tried everything I can think of from even to the extent of restarting my computer, and still no check box in the time tracking settings—images for reference.

![image](https://user-images.githubusercontent.com/52513636/115511216-d4b0e680-a23d-11eb-8930-a8dc1fb5f454.png)
![image](https://user-images.githubusercontent.com/52513636/115511320-f4e0a580-a23d-11eb-9d92-e2a82695b032.png)

**Comment by Wr4776** [comment]
I'm having the same issue. If anyone has a solution they could share, it would be much appreciated 👍

**Comment by ffalor** [comment]
Sorry I'm late to this. Looks like the removed this for some reason. Can you go into the Time Tracking plugin, and check the notifications bells?

![image](https://user-images.githubusercontent.com/35144141/130017197-23a54317-ec6a-42ad-b52c-4f9bd06535f9.png)

Let me know if this works for you, and I'll update the documentation.

**Comment by Wr4776** [comment]
I'm able to toggle the bells, so I think that fixes it right? I read the thread again but can't remember what the original issue was that I was having 🤦‍♂️

**Comment by ffalor** [comment]
Yes that _should_ fix it lol. I'll need to update the documentation to show this since it has changed from my original release.

**Comment by GenTEKS** [comment]
I think it has stopped working again.
